Ports/qemu/package.sh

```diff
@@ -12,8 +12,14 @@ configopts=(
     "--enable-sdl"
     "--enable-tcg"
     "--enable-tools"
+    "--enable-slirp"
+)
+depends=(
+  "glib"
+  "pixman"
+  "SDL2"
+  "libslirp"
```

Comment:
These should probably be in alphabetical order to prevent :redlinus: - also, single quotes.